R/whirl_r_session.R

```diff
@@ -230,6 +230,24 @@ wrs_check_status <- function(self, private, super) {
 wrs_log_script <- function(script, self, private, super) {
   private$current_script <- script
 
+  # Set the execute directory of the Quarto process calling the script
+  quarto_execute_dir <- options::opt("execute_dir", env = "whirl")
```

Comment:
so it means we can only change the option if we want to change the execute_dir. We can not for example do something like :
run("blabla", execute_dir = "example")
For me it is a bit restrictive but why not ^^